network-api/networkapi/templates/fragments/footer.html

```diff
@@ -32,7 +32,7 @@
                                  data-form-position="footer"
                                  data-button-text="{% trans "Sign up" context "Submit button for newsletter signup form" %}"
                                  data-cta-header="{% trans "Help shape the future of the web for the public good." %}"
-                                 data-cta-description="{% trans "<p>Join our <b>Mozilla News</b> email list to get action alerts & internet tips right to your inbox.</p>" %}"
+                                 data-cta-description="<p>{% trans "Join our <b>Mozilla News</b> email list to get action alerts & internet tips right to your inbox." %}</p>"
```

Comment:
Changed this to re-use the string from `take_action.html` signup form (which has already been translated)